bin/contest.py

```diff
@@ -35,7 +35,7 @@ def contest_yaml():
 
 def next_label(label):
```

Comment:
This is only used in skel.py now maybe move it there.

I was thinking: It would be cool to start at Z, and every time you add a problem all labels are shifted down by 1 and the new problem is appended. But maybe that's too complicated so just starting at X also works fine; we always have 3 problems anyway.

Reply:
Good idea :slightly_smiling_face: 

Yeah I was thinking about that too, but that would then require prepending problems to `problems.yaml` (instead of appending), that could become messy. :joy: I also wonder what would now happen if you would add a fourth problem to a test session (or, for that matter, a 27th problem to a regular contest). The label will probably become `'['` or something :stuck_out_tongue: But I assume we don't have to add corner cases for that, if it won't happen during regular use :innocent:

Reply:
Sounds good :slightly_smiling_face: 

I was also thinking about that, but I thought about prepending problems to `problems.yaml` (instead of appending them), so I didn't do it. Shifting down the labels would make sense, assuming that they follow the regular naming scheme. And any other cases probably fall outside of regular use, so we probably won't need to add corner cases for that. :innocent:

Reply:
Thinking about it more; I'd prefer to keep it simple (i.e. what you did now). This is all quite rare anyway, and it would significantly complicate the case. Also, labels like `[X1, X2, Y]` are possible, and shifting those down is not good.